app/views/account/meter_readings/_form.html.slim

```diff
@@ -0,0 +1,15 @@
+.content.pt-4
+  .form-component.form-component-meter-readings
+    = simple_form_for [:account, :utility_provider, @meter_reading], method: :post, :html=> { class: 'mx-auto billing-form' } do |f|
+      .form-group
+        = f.input :value, label: t('.meter_reading'),
+          input_html: { class: 'form-control' },
+          error_html: { class: 'form-error' }
+      .form-group.m-date
+        = f.input :date, label: t('.date'), as: :date, start_year: Date.today.year,
```

Comment:
I think user should not have possibility to set date
use DateTime.current

Reply:
done